README.md

```diff
@@ -30,6 +30,19 @@ dependencies {
 }
 ```
 
+Example apps
+------------
+
+You can find implementation example of the capture feature in the following modules:
+- `appcomponentapi` for Component API
+- `appscreenapi` for Screen API
+
+You can find an example for the payment feature in `appscreenapi` in `PayActivity`. 
+
+For testing the payment flow integration you can use the example app from [Business SDK](https://github.com/gini/gini-pay-business-sdk-android)
+for starting the flow.
+ 
+
```

Comment:
A few words about the client ids?
`You should have gotten client credentials for the Gini Pay Business SDK as well. You need to use those in the [Business SDK](https://github.com/gini/gini-pay-business-sdk-android) example app. Please contact us in case these client credentials are missing.`

Reply:
I changed the link to point to the example apps section in the readme. 
The section explains how to add the clientId & clientSecret.

Reply:
I see, thanks!